Orange/canvas/application/addons.py

```diff
@@ -399,6 +410,19 @@ def __init__(self, parent=None, **kwargs):
 
         self.layout().addWidget(buttons)
 
+        if not os.access(sysconfig.get_path("purelib"), os.W_OK):
+            if sysconfig.get_platform().startswith("macosx"):
+                info = "You must install Orange by dragging it into" \
+                       " Applications before installing add-ons."
+            else:
+                info = "Install destination is Read-only," \
+                       " therefore add-ons can not be installed."
```

Comment:
"You do not have permissions to write into Orange directory.\n" \
"You may need to contact an administrator for assistance."

To sporočilo je sicer malo trapasto, ker si očitno lahko tudi ti admin, ampak ne znam povedati krajše.